weave-scope-cloud-service/README.md

```diff
@@ -0,0 +1,91 @@
+
+##Running Weave Scope in Cloud Service Mode
+
+Once you've received a service-token id, use it to launch a Weave Scope probe on every machine that you want to monitor:
+
+~~~bash
++sudo wget -O /usr/local/bin/scope \
++  https://github.com/weaveworks/scope/releases/download/latest_release/scope
++sudo chmod a+x /usr/local/bin/scope
+sudo scope launch --service-token=<token>
+~~~
+
+>Note: If you are running Weave Net you can launch a Weave Scope probe onto one host and Weave Net, provided that you have weaveDNS running will discover the rest of the containers. 
+
+##Launching Weave Scope When Using Weave Net to Connect Containers
+
+If you are running Weave Scope on the same machine as the Weave network, the probe uses `weaveDNS` to automatically discover other apps and containers on your network. 
+
+Weave Scope registers itself using the address `scope.weave.local`. Each probe then, sends its report to every app registered under this address. In other words, if you are running Weave Net with DNS, you do not need to take any further steps. Launch once and Weave Net takes care of the rest. 
+
+If you don't want to use weaveDNS, you can instruct Weave Scope to cluster with other Weave Scope instances on the command line. Hostnames and IP addresses are acceptable, both with and without ports, for example:
+
+~~~bash
+scope launch scope1:4030 192.168.0.12 192.168.0.11:4030
+~~~
+
+Hostnames are regularly resolved as A records, where each answer is used as a target.
+
+
+##Visualizing Your Infrastructure
+
+Once Weave Scope is displayed in your browser, it gives you an immediate overview of your network, and shows all of your containerized apps. 
+
+An Overview of the App:![An Overview of the App](/guides/images/weave-scope/weave-scope-application-layers.png)
+
+Each circle on the map represents a container in your network. Circles connected by a line are containers communicating with each other. Use your mouse to explore the network. When you hover over a container or a connection Weave Scope highlights the connected containers to help you understand the topology quicker. This is especially helpful for infrastructures with many containers.
+
+If you are looking for more information about one of the database (db) containers, click on the container with the label tns_db1_1 and view the metrics about it.
+
+Metrics of the DB Container:![Details and Metrics of the DB Container](/guides/images/weave-scope/weave-scope-database-metrics.png)
+
+The panel on the right shows basic information about the container, for example the image and process names, as well as any network metrics, like ingress/egress byte rates and also the number of TCP connections.
+
+>Note: In this view, you can Pause, Restart, and Stop containers, and for this reason, access to your Weave Scope infrastructure visualization should be restricted to trusted individuals. 
```

Comment:
"your Weave Scope infrastructure visualization" seems like a mouthful, when you could just say "Weave Scope"